cohortextractor/patients.py

```diff
@@ -2858,3 +2858,73 @@ def with_an_ons_cis_record(
             )
     """
     return "with_an_ons_cis_record", locals()
+
+
+
+def with_record_in_UKRR(
+    # picks dataset held by UKRR
+    from_dataset,
+    returning,
```

Comment:
pretty much everywhere else in patients.py we use keyword args only - I'm assuming that's to prevent users putting positional args in the wrong order

Reply:
I'm not sure I completely understand this comment. Do you mean change the order of them or remove? We need to be able to identify the dataset in the UKRR somehow.

Reply:
I just meant to make them keywords instead of positional args.  It means users can't mix up the order and specify `returning` where we think we're getting `from_dataset`.  

Like:
```
def with_record_in_UKRR(
    # picks dataset held by UKRR
    from_dataset=None,
    returning="binary_flag",
    ...
```

I think elsewhere we generally default to `binary_flag` for `returning`.  `from_dataset` could have a default if it makes sense to default to one of them, or None and we raise an exception if they don't provide it